docs/kyma/docs/04-03-local-installation-from-sources.md

```diff
@@ -17,14 +17,16 @@ To start the local installation, run the following command:
 ./installation/cmd/run.sh
 ```
 
-This script sets up default parameters, starts Minikube, builds the Kyma-Installer, generates local configuration, creates the Installation custom resource, and sets up the Installer.
+This script sets up default parameters, starts Minikube, builds the Kyma Installer, generates local configuration, creates the Installation custom resource, and sets up the Installer.
 
-> **NOTE:** See the **Local installation scripts** document for a detailed explanation of the `run.sh` script and the subscripts it triggers.
+> **NOTE:** See [this](#installation-local-installation-scripts-deep-dive) document for a detailed explanation of the `run.sh` script and the subscripts it triggers.
 
 You can execute the `installation/cmd/run.sh` script with the following parameters:
 
+- `--password {YOUR_PASSWORD}` which allows you to set a password for the **admin@kyma.cx** user.
 - `--skip-minikube-start` which skips the execution of the `installation/scripts/minikube.sh` script.
 - `--vm-driver` which points to either `virtualbox` or `hyperkit`, depending on your operating system.
 
-Read the **Reinstall Kyma** document to learn how to reinstall Kyma without deleting the cluster from Minikube.
-To learn how to test Kyma, see the **Testing Kyma** document.
+Read [this](#installation-reinstall-kyma) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
+
+To learn how to test Kyma, see [this](#installation-reinstall-kyma) document.
```

Comment:
Are you sure it's the right link? It lead to `Reinstall Kyma`, not to any testing scenario.